webapp/src/ts/services/contact-muted.service.ts

```diff
@@ -8,28 +8,31 @@ export class ContactMutedService {
 
   constructor() { }
 
-  getMuted(doc, lineage?) {
+  getMutedParent(doc, lineage?) {
```

Comment:
This is a bit misleading because it can return `doc` not a parent. Maybe `getMutedDoc` is a better name?